docs/SRS/SRS.tex

```diff
@@ -640,72 +640,72 @@ \subsection{Functional Requirements}
-\textbf{Fit Criterion:} The system should provide relevant budgeting suggestions based on the user's purchase history and budgeting preferences.\\
+\textbf{Description:} The system must analyze the user's purchase history and compare it to the budgeting preferences the user has submitted. It must also alert or recommend cheaper grocery alternatives when the user exceeds their budget.\\
+\textbf{Rationale:} This functionality allows users to make informed decisions and encourages them towards making well informed grocery purchases.\\
+\textbf{Fit Criterion:} Suggestions should be considered relevant and useful in user feedback surveys, with at least 70\% positive response rate.\\
 
 \medskip
 \noindent \textbf{Requirement ID:} FR9\\
 \textbf{Use Case:} 12\\
-\textbf{Description:} When a recently purchased item by the user is found at a lower price, the system should prompt the user with information about the item, its price, and the location where it is available at a lower cost.\\
-\textbf{Rationale:} This requirement enhances user experience by helping users save money by identifying opportunities for cost savings.\\
-\textbf{Fit Criterion:} The system should prompt the user with information about cheaper alternatives when available within the user's location preferences.\\
+\textbf{Description:} The system must actively monitor prices of items that a user has purchased within the last 10 days. When it detects that an item is available at a lower price, it should promptly notify the user.\\
+\textbf{Rationale:} Notifying users about lower price alternatives on grocery items they recently purchased enlightens users about potential cost saving alternatives and emphasizes the usefulness of the application.\\
+\textbf{Fit Criterion:} Notifications on lower price alternatives must be delivered to the user within 24 hours of detection.\\
 
 \medskip
 \noindent \textbf{Requirement ID:} FR10\\
 \textbf{Use Case:} 10\\
-\textbf{Description:} The system must retrieve and display a visual representation of the user's purchase history, including details on items purchased and their respective timestamps.\\
-\textbf{Rationale:} This requirement enables users to monitor their spending habits and make informed financial decisions by providing a clear view of their purchase history.\\
-\textbf{Fit Criterion:} The system should present the user's purchase history accurately and in a user-friendly manner.\\
+\textbf{Description:} The system must fetch and display a visual representation of the user's grocery purchase history, including details on items purchased and their respective timestamps.\\
+\textbf{Rationale:} Providing a visual representation of grocery purchase history converts unprocessed data into a comprehensive and easily understandable format for users.\\
+\textbf{Fit Criterion:} The system should present the user's purchase history data with over 95\% accuracy.\\
```

Comment:
For this line, I think it should be 99/% based on NFRs which is the current value set there. I think analytics in particular should be **very** accurate considering we're dealing with financials. So I think we need to set a high bar of accuracy and low error with rounding/exact numbers.